starter-code/index.html

```diff
@@ -2,41 +2,76 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Spotify</title>
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+  <div class="top-bar">
+    <img class="green-logo" src="./images/spotify-logo.png" alt="green spotify logo">
+    <nav>
+      <ul class="nav-list">
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </nav>
+  </div>
+
+<div class="hero-section">
+  <img class="landing-image"src="./images/landing.jpg" alt="landing page image">
+  <h1 class="main-heading">Music for everyone.</h1>
+  <h2 class="sub-heading">Spotify is now free on mobile, tablet and computer.</h2>
+  <h2 class="sub-heading">Listen to the right music, wherever you are.</h2>
+</div>
+
+<div class="center-section">
+
+  <h3 class="center-section-heading">What’s on Spotify?</h3>
+
+    <div class="center-section-group"></div>
```

Comment:
bottom 3 divs would go into this div if you are planning on modifying the 3 together for positioning.